trello/features/acceptance/BoardAcceptance.feature

```diff
@@ -4,23 +4,21 @@ Feature: Board
 
 
   Scenario: Changes the name of the board
-    Given Sets a POST request to "/boards/"
-    And Sets the following Path Params:
+    Given Sets a "POST" request to "/boards/"
       | key  | value        |
       | name | GherkinBoard |
     And Sends request
     And Should return status code 200
     And Saves response as "BoardObject"
-    And Sets a PUT request to /boards/"(BoardObject.id)"
-    And Sets the following Path Params
+    And Sets a "PUT" request to /boards/"(BoardObject.id)"
       | key     | value            |
       | name    | UpdateBoardName  |
       | idBoard | (BoardObject.id) |
     When Sends request
     Then Should return status code 200
     And Validates response body
     And Validates schema
-    And Sets a DELETE request to /boards/"(BoardObject.id)"
+    And Sets a "DELETE" request to /boards/"(BoardObject.id)"
```

Comment:
this line of code add to table  **/boards/"(BoardObject.id)"**

Reply:
It would be 'Sets a "DELETE" request to "/boards/"'